packages/x-components/src/x-modules/facets/store/actions/set-filters-from-url.action.ts

```diff
@@ -0,0 +1,29 @@
+import { createRawFilters, Dictionary } from '../../../../utils';
+import { UrlParamValue } from '../../../url';
```

Comment:
Avoid importing from barrels. It is very easy to create cyclical dependencies that can result in errors at runtime. In this case you are even importing from a barrel that should be exporting this action, which is worse.